config/dispatcher/dispatcher.ex

```diff
@@ -650,14 +652,44 @@ defmodule Dispatcher do
     forward conn, path, "http://berichtencentrum-melding/melding"
   end
 
+  #################################################################
+  # Loket
+  #################################################################
+
+  # Login
+
+  match "/sessions/*path", %{ reverse_host: ["loket" | _rest] } do
+    forward conn, path, "http://login-loket/sessions/"
+  end
+
+  # Frontend
+
+  get "/assets/*path",  %{ reverse_host: ["loket" | _rest] }  do
+    forward conn, path, "http://loket/assets/"
+  end
+
+  get "/@appuniversum/*path", %{ reverse_host: ["loket" | _rest] } do
+    forward conn, path, "http://loket/@appuniversum/"
+  end
+
+  match "/*_path", %{ reverse_host: ["loket" | _rest] } do
+    # *_path allows a path to be supplied, but will not yield
+    # an error that we don't use the path variable.
+    forward conn, [], "http://loket/index.html"
```

Comment:
Does this actually work already? The loket frontend still uses the old proxy-service based setup which expects to be the entry point.

Reply:
Yes, this fully works. At least for me locally. The backend on the proxy service references the identifier. I assume the proxy service then just forwards requests exactly like they come in, virtual host information included, but the dispatcher will look at the MIME types to correctly forward all requests.

Reply:
Fair enough :+1: 

We should probably just change the base image in the frontend though, since it's a quick change and we were planning on doing that anyways. I'll create a PR.